### ISSUE TITLE
Node version?

### ISSUE THREAD

**Issue by reinethernal** [daxcay/ComfyUI-TG#5]
Which node version i need to use? 
I have v23.4.0 installed, but i receive errors:
    at /root/ComfyUI/node_modules/node-telegram-bot-api/src/telegramPolling.js:106:17
    at tryCatcher (/root/ComfyUI/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:547:31)
    at Promise._settlePromise (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:604:18)
    at Promise._settlePromise0 (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:649:10)
    at Promise._settlePromises (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:729:18)
    at _drainQueueStep (/root/ComfyUI/node_modules/bluebird/js/release/async.js:93:12)
    at _drainQueue (/root/ComfyUI/node_modules/bluebird/js/release/async.js:86:9)
    at Async._drainQueues (/root/ComfyUI/node_modules/bluebird/js/release/async.js:102:5)
    at Async.drainQueues [as _onImmediate] (/root/ComfyUI/node_modules/bluebird/js/release/async.js:15:14)
    at process.processImmediate (node:internal/timers:511:21)
From: ReinEthernal Command: /s 240 0 start Type: text
Error reading file: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at Object.join (node:path:1261:7)
    at readJSONFile (/root/ComfyUI/custom_nodes/ComfyUI-TG/nodejs/project/app.js:161:24)
    at setCommand (/root/ComfyUI/custom_nodes/ComfyUI-TG/nodejs/project/app.js:431:35)
    at TelegramBot.<anonymous> (/root/ComfyUI/custom_nodes/ComfyUI-TG/nodejs/project/app.js:553:4)
    at TelegramBot.emit (/root/ComfyUI/node_modules/eventemitter3/index.js:182:35)
    at TelegramBot.processUpdate (/root/ComfyUI/node_modules/node-telegram-bot-api/src/telegram.js:709:14)
    at /root/ComfyUI/node_modules/node-telegram-bot-api/src/telegramPolling.js:110:22
    at Array.forEach (<anonymous>)
    at /root/ComfyUI/node_modules/node-telegram-bot-api/src/telegramPolling.js:106:17
    at tryCatcher (/root/ComfyUI/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:547:31)
    at Promise._settlePromise (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:604:18)
    at Promise._settlePromise0 (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:649:10)
    at Promise._settlePromises (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:729:18)
    at _drainQueueStep (/root/ComfyUI/node_modules/bluebird/js/release/async.js:93:12)
    at _drainQueue (/root/ComfyUI/node_modules/bluebird/js/release/async.js:86:9)
    at Async._drainQueues (/root/ComfyUI/node_modules/bluebird/js/release/async.js:102:5)
    at Async.drainQueues [as _onImmediate] (/root/ComfyUI/node_modules/bluebird/js/release/async.js:15:14)
    at process.processImmediate (node:internal/timers:511:21) {
  code: 'ERR_INVALID_ARG_TYPE'
}
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at setCommand (/root/ComfyUI/custom_nodes/ComfyUI-TG/nodejs/project/app.js:445:32)
    at TelegramBot.<anonymous> (/root/ComfyUI/custom_nodes/ComfyUI-TG/nodejs/project/app.js:553:4)
    at TelegramBot.emit (/root/ComfyUI/node_modules/eventemitter3/index.js:182:35)
    at TelegramBot.processUpdate (/root/ComfyUI/node_modules/node-telegram-bot-api/src/telegram.js:709:14)
    at /root/ComfyUI/node_modules/node-telegram-bot-api/src/telegramPolling.js:110:22
    at Array.forEach (<anonymous>)
    at /root/ComfyUI/node_modules/node-telegram-bot-api/src/telegramPolling.js:106:17
    at tryCatcher (/root/ComfyUI/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:547:31)
    at Promise._settlePromise (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:604:18)
    at Promise._settlePromise0 (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:649:10)
    at Promise._settlePromises (/root/ComfyUI/node_modules/bluebird/js/release/promise.js:729:18)
    at _drainQueueStep (/root/ComfyUI/node_modules/bluebird/js/release/async.js:93:12)
    at _drainQueue (/root/ComfyUI/node_modules/bluebird/js/release/async.js:86:9)
    at Async._drainQueues (/root/ComfyUI/node_modules/bluebird/js/release/async.js:102:5)
    at Async.drainQueues [as _onImmediate] (/root/ComfyUI/node_modules/bluebird/js/release/async.js:15:14)
    at process.processImmediate (node:internal/timers:511:21)
^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[A^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^Z
[1]+  Stopped                 python3 main.py --listen 192.168.129.54
(venv) root@sdwebui:~/ComfyUI# node -v
v23.4.0

**Comment by daxcay** [comment]
How this error occurred,
what did you do exactly,

Is this the first time; just after install ?

Join My discord so we can solve it in call if possible.

https://discord.com/invite/UyGkJycvyW

**Comment by reinethernal** [comment]
Thank you! Just joined)

**Comment by reinethernal** [comment]
The error was not that I thought I had installed the wrong version of node, but that I was running it on a remote lxc container instead of on the local one. To access the download interface I had to go to the container address (i.e. remote-host-ip:4001) and the menu opened. After that it was only necessary to follow further instructions. 
![image](https://github.com/user-attachments/assets/5ec7f26b-9ed9-49a2-8d43-92345ff9a102)


Thank you very much @daxcay !
